eng/Versions.props

```diff
@@ -208,5 +208,9 @@
     <!-- pinned dependency. This package is not being produced outside of the 2.0 branch of corefx and should not change. -->
     <CLI_NETStandardLibraryNETFrameworkVersion>2.0.1-servicing-26011-01</CLI_NETStandardLibraryNETFrameworkVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <!-- pinned dependency. This package is not being produced outside of the 2.0 branch of corefx and should not change. -->
+    <DotNetDeploymentReleasesPackageVersion>1.0.0-preview5.1.22263.1</DotNetDeploymentReleasesPackageVersion>
```

Comment:
This introduced a prebuilt in source-build.  I don't know what the best way to resolve this is.  This version number needs to be in sync with the Versions.Details.xml.  This is what tells source-build which version to build.

Reply:
Hmm, yeah, we'd need to add this to the versions.props and the versionsdetails.xml to allow the SDK value to flow to install I think

Reply:
Can you handle updating the sdk repo @joeloff.  This work feels like it is covered by https://github.com/dotnet/sdk/issues/20262

Reply:
I opened an installer PR to address this - https://github.com/dotnet/installer/pull/14176